packages/webview_flutter/webview_flutter_android/pigeons/android_webkit.dart

```diff
@@ -513,7 +582,7 @@ abstract class WebChromeClient {
   late List<String> Function(
     WebView webView,
     FileChooserParams params,
-  )? onShowFileChooser;
+  ) onShowFileChooser;
```

Comment:
Why the change to non-nullable on these callbacks?

Reply:
This was caused by https://github.com/flutter/packages/pull/8567. They are already implemented in the `AndroidWebViewController`: https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart#L187. The method just wasn't required on the Dart side.